src/main/java/org/opensearch/neuralsearch/query/NeuralKNNQueryBuilder.java

```diff
@@ -107,6 +109,11 @@ public Builder rescoreContext(RescoreContext rescoreContext) {
             return this;
         }
 
+        public Builder originalQueryText(String originalQueryText) {
```

Comment:
Please add javadoc. We already have an issue for reducing the warning logs https://github.com/opensearch-project/neural-search/issues/1095

Reply:
If possible please add the javadoc on all public methods of this class.